.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -673,3 +673,20 @@ cross_ResilientService:
       DescribeLoadBalancers, CreateTargetGroup, DescribeTargetGroups, DeleteTargetGroup,
       CreateLoadBalancer, CreateListener, DeleteLoadBalancer, DescribeTargetHealth}
     iam: {CreateInstanceProfile, DeleteInstanceProfile}
+cross_FMPlayground:
+  title: Create a Foundation Model Playground application that offers different playgrounds to interact with foundation models via &BR; API
```

Comment:
Suggest adding acronym, since its used
Create a Foundation Model (FM) Playground

Reply:
I've applied the fix